docs/source/markdown/podman-volume-create.1.md

```diff
@@ -129,6 +129,80 @@ All containers are assigned larger project IDs (e.g. >= 100000).
 All volume assigned project IDs larger project IDs starting with 200000.
 This prevents xfs_quota management conflicts with containers/storage.
 
+## MOUNT EXAMPLES
+
+podman volume create allows you to pass the `type`, `device`, and `o` options to `mount(8)` when using the `local` driver.  
+
+Here are some examples of what you can do with that
+
+## [s3fs-fuse](https://github.com/s3fs-fuse/s3fs-fuse)
+
+[s3fs-fuse](https://github.com/s3fs-fuse/s3fs-fuse) or just `s3fs`, is a fuse filesystem that allows you to mount s3 prefixes as filesystem mounts.
+
+**Installing:**
+```shell
+dnf install s3fs-fuse
```

Comment:
Always include a prompt. Always. In this case, prompt should be `#` to indicate that the command must be run as root. Alternatively, use `$ sudo ...`.